tree/NetworkFetcher/Model/TrendDays.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  TrandDays.swift
+//  tree
+//
+//  Created by ParkSungJoon on 29/01/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import Foundation
+
+public struct TrendDays: Codable {
+    public let trend: Default
+    private enum CodingKeys: String, CodingKey {
+        case trend = "default"
+    }
+}
+
+public struct Default: Codable {
+    public let searchesByDays: [TrendingSearchesDay]
+    public let endDateForNextRequest: String
+    public let rssFeedPageURL: String
+    private enum CodingKeys: String, CodingKey {
+        case endDateForNextRequest
+        case rssFeedPageURL = "rssFeedPageUrl"
+        case searchesByDays = "trendingSearchesDays"
+    }
+}
+
+public struct TrendingSearchesDay: Codable {
+    public let date, formattedDate: String
+    public let keywordList: [TrendingSearch]
```

Comment:
배열 변수의 경우 -List 접미사를 붙이기보다는 복수형으로 사용하는것이 좀 더 스위프트스러운 방식입니다.
```suggestion
    public let keywords: [TrendingSearch]
```